### PR TITLE
LPS-165583 Fixed path to export page.

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/ExportPage.tsx [hunk 1]
```diff
@@ -69,14 +60,24 @@ const ExportPage = () => {
...
-					/>
+					<ClayButton.Group spaced>
+						<Link
+							className={classNames('btn btn-primary', {
+								disabled: !caseIds.length,
+							})}
+							target="_blank"
+							to={`/export/${id}`}
+						>
+							{i18n.translate(i18n.translate('export'))}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/ExportPage.tsx:71]:
```suggestion
							{i18n.translate('export')}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
1. i18n.translate duplicate
2. Add translation to en-us because probably not exist

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**